client/src/components/BottomPanel.tsx

```diff
@@ -220,11 +219,7 @@ const PanelContent = ({
 			return <WhiteboardPanel setBrushColor={setBrushColor} />;
 		case 'settings':
 			return (
-				// <Profile />
-				<SettingsPanel
-					onSubmitUrl={(url) => onAction('settings', 'url', url)}
-					onChangeName={(name) => onAction('settings', 'name', name)}
-				/>
+				<div></div>
```

Comment:
I think we can remove this, similarly to `new-room` we dont have to render any bottom panel if we dont want to. Additionally, you can render `null` instead of an empty div for more clarity and the same result